items.json

```diff
@@ -59062,19 +59062,19 @@
         "thrustSpeed": 89,
         "swingDamage": 30,
         "swingDamageType": "Blunt",
-        "swingSpeed": 77
+        "swingSpeed": 76
       }
     ]
   },
   {
-    "id": "crpg_fine_steel_mace_v1_h3",
-    "baseId": "crpg_fine_steel_mace_v1",
+    "id": "crpg_fine_steel_mace_v2_h3",
```

Comment:
example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.